src/hbase/test-util/mini-cluster.cc

```diff
@@ -34,7 +34,7 @@ JNIEnv *MiniCluster::CreateVM(JavaVM **jvm) {
   char *classpath = getenv("CLASSPATH");
   std::string clspath;
   if (classpath == NULL || strstr(classpath, "-tests.jar") == NULL) {
-    std::string clsPathFilePath("../../../hbase-build-configuration/target/cached_classpath.txt");
+    std::string clsPathFilePath("../target/cached_classpath.txt");
```

Comment:
I reckon this doesn't work after removing the native client out of the "main" HBase java repository.